fact-bounty-client/src/pages/notFound/styles/index.css

```diff
@@ -8,7 +8,7 @@
   margin: 4em;
 }
 
-img {

```

Comment:
I added styles using img(element selector) because its precedence is lower than class and id so in future if one needs to override the styles it can use class or id. But now as you replaced it with the class we will need a selector which is higher on the precedence table to override CSS.

Reply:
@kartikpandey2 When you are using the img element for styling, the style is being added to every img element, which may break the styling of other elements, for example in the case of the image in the news items, the height got changed due to it.

Reply:
Oh, I didn't know that img styles I added were conflicting with story images. Thanks for the correction 😃.

Reply:
:thumbsup: